### DISCUSSION TITLE
Is there a way to copy / paste cargo / technology section?

### DISCUSSION THREAD

**Discussion by ascornpire** [zencq/NomNom#184] [Q&A]
Wondering if this is planned, considered, or wanted to be able to copy the cargo/technology from one ship to another?

**Comment by Droid8Apple** [comment]
> Wondering if this is planned, considered, or wanted to be able to copy the cargo/technology from one ship to another?

Same - it's the only thing in life I've ever wanted from the save editors. Goatfungus was basically a lite version of this one... just as tedious, having to do 17,000 clicks just to add a single item. You would think, given the rudimentary nature of it, that if delete works when hitting the button - CTRL+C and CTRL+V would also be easy. Especially considering that you can make backups... so let us back up *only* a specific section and not all the seeds and colors and stats and stuff. Just technology; I don't even mind cargo not being there.

If you found any way, please let me know. After so many years, there's nothing I hate more than setting up a new ship. And we get one every quarter.

**Reply by ascornpire** [reply]
I did not, but my hope runs deep, into cope most likely.

**Reply by Droid8Apple** [reply]
I did it lol. Have you noticed when clicking "my collection" it has all ships (including twitch drops you've never claimed?) - well it'll show each ship you backup of your own as well. 

So I backed up my exotic ship (named Zephyr Prime). It has everything all set up as I have had it since those were the coolest ships. Next, I changed it to the Starborn Runner from the "ship type" drop down and generated a seed for it, saved and reloaded and there it was - a second Starborn Runner except it had *everything* the exotic had in its cargo, stats, and technology. I aptly named it "Modded Runner" and loaded up the game and it was there perfectly. Same paint as the one I claimed from the expedition and everything. If you had a regular fighter though, for example, you'd just copy the stats and seed of the one you want to be modded into the field of your recently backed up ship you're using as a template. Boom. Is it as easy as a damned copy and paste? Nope lol. Did it work? Yep. And if you're not full of ships you wouldn't even need to delete it. The "collection"  (small icon near where you'd upload a picture of your ship) is anything you had backed up so you can always pull ships in and out on a whim. 

Really sorry if that's not making sense - it's late and I'm running on fumes. But I can make a quick video in the morning if that'd help. Least I could do for the only other person I've seen ask this since 2016.
